src/server/game/Server/WorldSocket.cpp

```diff
@@ -689,8 +689,8 @@ void WorldSocket::HandleAuthSessionCallback(std::shared_ptr<WorldPackets::Auth::
     if (memcmp(hmac.GetDigest(), authSession->Digest.data(), authSession->Digest.size()) != 0)
     {
         TC_LOG_ERROR("network", "WorldSocket::HandleAuthSession: Authentication failed for account: %u ('%s') address: %s", account.Game.Id, authSession->RealmJoinTicket.c_str(), address.c_str());
-        DelayedCloseSocket();
-        return;
+        //DelayedCloseSocket();
```

Comment:
why?

Reply:
Because I don't have the ClientTypeSeed_Wn64 to verify the password hash.